docs/getting-started/reverting-atlas.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Reverting Atlas
+description: How to uninstall AtlasOS
+icon: material/emoticon-cry
+---
+
+# :material-emoticon-cry: Reverting Atlas
+
+As Atlas disables the 'Reset this PC' feature due to its unreliability and not being a true reinstall of Windows, some users get confused about how to get back to stock Windows.
+
+This page details how to reinstall from Atlas to stock Windows fully, which uninstalls Atlas.
+
+## :material-handshake: Before uninstalling
+
+If you're experiencing issues, ensure you're on the [latest version of Atlas](installation.md), and then consider getting help from the following:
+
+- :material-file-search: The documentation you're on currently
+- <a onclick="clickKapaAi();" href="javascript:void(0)">
+    <p>:material-robot: 'Ask AI' in the corner of this page</p>
```

Comment:
paragraph tag is causing formatting issue

Reply:
https://github.com/Atlas-OS/docs/assets/65787561/7f749817-09a9-4c23-ae5d-dba89d4ce563

Where?

Reply:
As shown on desktop
![image](https://github.com/Atlas-OS/docs/assets/83741557/54d7d471-ec43-4489-85ec-2ccf641fb2c6)

Reply:
Which browser?

Reply:
Currently using firefox developer edition (latest version)